Cargo.toml

```diff
@@ -79,6 +79,7 @@ crossbeam = "0.2"
 fxhash = "0.2"
 derive_more = "0.11.0"
 num = "0.2.0"
+hex="0.3.2"
```

Comment:
Please add space like `hex = "0.3"`, and also I think it is not necessary to specify the minor version.